src/devices/elko.ts

```diff
@@ -197,6 +197,48 @@ const definitions: DefinitionWithExtend[] = [
             device.save();
         },
     },
+    {
+        zigbeeModel: ['NHMOTION/UNIDIM/1'],
+        model: 'EKO06984',
+        vendor: 'ELKO',
+        description: 'SmartPir with push dimmer',
+        fromZigbee: [fz.on_off, fz.brightness, fz.illuminance, fz.level_config, fz.schneider_lighting_ballast_configuration, fz.occupancy],
```

Comment:
Can you try with modern extend? E.g. `extend: [light({configureReporting: true})],`